### PR TITLE
Feat/#302 endpoint

### PR DESCRIPTION
**Pull request by ciaranschutte**
merge after: https://github.com/icgc-argo/donor-submission-aggregator/pull/307
- puts "indexing" route behind feature flag
- moves "indexing" route into it's own file

### CODE REVIEW

**Review by joneubank — COMMENTED** [review]
I think this implementation works. I'll approve once the preceding PR gets merged and this one just shows the changes related to the feature flag.

File: src/utils.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@
  * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
```
Comment (ciaranschutte) [inline src/utils.ts:16]: formatting

File: .schema.env [hunk 2]
```diff
@@ -54,6 +54,7 @@ EGO_PUBLIC_KEY=-----BEGIN PUBLIC KEY-----\r\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMII
 
 
 
+
```
Comment (justincorrigible) [inline .schema.env:57]: No need for a fourth empty line
```suggestion
```

**Review by justincorrigible — APPROVED** [review]
nice. squash away!

**Pull request merged**